index.html

```diff
@@ -62,28 +62,34 @@ <h1 class="mainHeaderTitle">You think you are smart? Prove it!</h1>
         <nav>
           <ul class="navbar" style="list-style: none">
             <li>
-              <a href="index.html"
-                ><img
-                  class="navIcon1"
-                  src="assets/home-5591299_1280.png"
-                  alt="this is the home button"
-              /></a>
+              <button aria-label="Close" onclick="{}">
+                <a href="index.html"
+                  ><img
+                    class="navIcon1"
+                    src="assets/home-5591299_1280.png"
+                    alt="this is the home button"
+                /></a>
+              </button>
             </li>
             <li>
-              <a href="bookmarks.html"
-                ><img
-                  class="navIcon2"
-                  src="assets/bookmark-6751601_1280.png"
-                  alt="This is the bookmark button"
-              /></a>
+              <button aria-label="Close" onclick="{}">
```

Comment:
the alt tag should tell the user what they can do here, for example "go to bookmarks section"